tests/sles4sap/patterns.pm

```diff
@@ -31,6 +31,13 @@ sub run {
 
     my $base_pattern = is_sle('>=15') ? 'patterns-server-enterprise-sap_server' : 'patterns-sles-sap_server';
 
+    # In case of DVD based installation we need to (re-)enable the local repos
+    # Non-SCC installation usually means installation from DVD
+    assert_script_run 'zypper mr -e -l'
+      if is_sle('15+')
+      and !check_var('SCC_REGISTER', 'installation')
+      and get_var('ISO_1');
```

Comment:
Shouldn't we check the contents of `ISO_1` to be sure that the string it contains is the Packages-DVD?

Reply:
We can yes, I was not sure in case of change in the name but we can see this later if it happens.